applications/tari_merge_mining_proxy/src/config.rs

```diff
@@ -41,6 +42,8 @@ pub struct MergeMiningProxyConfig {
     pub base_node_grpc_address: Multiaddr,
     /// The Tari console wallet's GRPC address
     pub console_wallet_grpc_address: Multiaddr,
+    /// GRPC authentication for console wallet
+    pub console_wallet_grpc_authentication: GrpcAuthentication,
```

Comment:
Please update the `common/config/presets/f_merge_mining_proxy.toml` file with the new setting